catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -74,6 +74,26 @@ const SearchForm = ({
 
   function updateUrl(unfilteredSearchResults, form) {
```

Comment:
Could we start typing these functions to make it easier to know what we're getting (and make it type safe).